backend/kaggle/datasets_manager.py

```diff
@@ -0,0 +1,104 @@
+import os
+import csv
+from os import path
+from pathlib import Path
+
+DEFAULT_DOWNLOAD_PATH = path.join(Path.home(), Path('.kaggle/'))
+
+
+def download(dataset, download_path=DEFAULT_DOWNLOAD_PATH):
+    """Download a dataset from kaggle then save it to a directory.
+    Parameters:
+        dataset : a string.
+        download_path : a string or any Path object.
+
+    Return:
+        dataset_path : full folder path where dataset is downloaded.
+        None : if dataset is already existed in download path.
+
+    Example:
+        This is a link to a kaggle dataset:
+        https://www.kaggle.com/aungpyaeap/tictactoe-endgame-dataset-uci
+
+        dataset = 'aungpyaeap/tictactoe-endgame-dataset-uci'
+        download_path = '~/.kaggle/'
+        dataset_path = download(dataset, download_path)
+
+        dataset_path = '~/.kaggle/aungpyaeap/tictactoe-endgame-dataset-uci/'
+    """
+    try:
+        os.mkdir(path.join(download_path, Path(dataset.split('/')[0])))
+        dataset_path = path.join(download_path, Path(dataset))
+        os.mkdir(dataset_path)
+
+    except FileExistsError:
+        return None
+
+    os.system(f'kaggle datasets download {dataset} -p {dataset_path} --unzip')
+    return dataset_path
+
+
+class skin_cancer_mnist:
```

Comment:
One last thing: Class name should be written in PascalCase.
```suggestion
class SkinCancerMnist:
```